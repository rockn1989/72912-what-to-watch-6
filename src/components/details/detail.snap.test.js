import React from 'react';
import {render} from '@testing-library/react';
import Details from './details';

it(`Should Details render correctly`, () => {
  const film = {
    released: 13,
    genre: `Comedy`,
    runTime: `13/08/1999`,
    director: `J.J. Abrams`,
    starring: [`Alena`, `Oleg`]
  };

  const {container} = render(<Details film={film}/>);
  expect(container).toMatchSnapshot();
});
