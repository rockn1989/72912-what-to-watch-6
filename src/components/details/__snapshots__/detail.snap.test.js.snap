// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Details render correctly 1`] = `
<div>
  <div
    class="movie-card__text movie-card__row"
  >
    <div
      class="movie-card__text-col"
    >
      <p
        class="movie-card__details-item"
      >
        <strong
          class="movie-card__details-name"
        >
          Director
        </strong>
        <span
          class="movie-card__details-value"
        >
          J.J. Abrams
        </span>
      </p>
      <p
        class="movie-card__details-item"
      >
        <strong
          class="movie-card__details-name"
        >
          Starring
        </strong>
        <span
          class="movie-card__details-value"
        >
          Alena, 
          Oleg
        </span>
      </p>
    </div>
    <div
      class="movie-card__text-col"
    >
      <p
        class="movie-card__details-item"
      >
        <strong
          class="movie-card__details-name"
        >
          Run Time
        </strong>
        <span
          class="movie-card__details-value"
        >
          NaNh NaNm
        </span>
      </p>
      <p
        class="movie-card__details-item"
      >
        <strong
          class="movie-card__details-name"
        >
          Genre
        </strong>
        <span
          class="movie-card__details-value"
        >
          Comedy
        </span>
      </p>
      <p
        class="movie-card__details-item"
      >
        <strong
          class="movie-card__details-name"
        >
          Released
        </strong>
        <span
          class="movie-card__details-value"
        >
          13
        </span>
      </p>
    </div>
  </div>
</div>
`;
